app.js

```diff
@@ -1,8 +1,11 @@
 const express = require("express")
 const app = express()
-const {getApi,getApiTopics,getApiArticleById, getApiArticles, getApiArticleComments}= require("./controllers/api.controller")
+const {getApi,getApiTopics,getApiArticleById, getApiArticles, getApiArticleComments,postArticleComment }= require("./controllers/api.controller")
+
 const {customErrorHandler,serverErrorHandler,postgresErrorHandler}= require("./errors")
 
+app.use(express.json())
```

Comment:
Really good demonstration of TDD principles to include this only _now that we need it_ to pass the test